zoom/plugin.spec.yaml

```diff
@@ -355,3 +355,50 @@ triggers:
         description: User Activity
         type: user_activity
         required: false
+events:
+  user_activity_event:
+    title: User Activity Event
+    description: Retrieve user activity events
+    input:
+      activity_type:
+        title: Activity Type
+        description: Type of user activity to match event
+        type: string
+        required: true
+        enum:
+        - "Sign in"
+        - "Sign out"
+        - "All"
+        example: "All"
+    output:
+      user_activity:
```

Comment:
@jschipp-r7 Recommended perhaps a "state" true/false on outputs to track, but I think this might be too limiting. My thoughts are that while you have a single "state" at the end of a run, you ultimately could have thousands of "events" returned.  Also seems like the output (a user specific thing) could be confusing if we start adding in variables that don't matter to the end user.  My two cents.

Reply:
What if we supported both? easy cases you set variable to true. More complex data, you define your own? Just a thought

Reply:
I'm open to this tbh.  We could find a way to support both.  If `store_state:true` on the particular output field then we include that with our helpers to define state.  Thoughts?

Reply:
sounds good to me, 

I love one-word things but this is just my OCPD: `state: true`

Reply:
yeah, was trying to do that without it being confusing but we can move it to `state: true`.  I ended up on `persist_state: true`.  Will update